idaes/models/properties/tests/test_harness.py

```diff
@@ -74,66 +74,76 @@ def frame(self):
 
         m.fs.params = self.prop_pack(default=self.param_args)
 
+        m.fs.props = m.fs.params.build_state_block(
```

Comment:
Was this moved solely so that `test_state_block_construction` would have a shorter runtime? Looks okay to me, just wondering why it was moved.

Reply:
No, this was moved as this code was added in a `unit` test but was also required for `component` tests. Thus,, component tests would fail if the unit tests were not run at the same time.